dynadjust/dynadjust/dnaadjust/dnaadjust-multi.cpp

```diff
@@ -768,7 +768,7 @@ void adjust_prepare_thread::operator()()
 	boost::shared_ptr<thread> p;
```

Comment:
I agree Neil - good to make the namespaces explicit, wherever possible.

On your aside comment, I was wondering whether we should consider moving from boost to std, whenever functions from boost have become part of the C++ Standard.  For example, thread is now part of std.  In this context, would it be wise to consider trailling std::thread and test for any breaks?  This might create more work than we're hoping for.

I don't have a strong opinion on this and would be happy to be guided by others who know more about the boost to std migration practice.

Reply:
I think it is a good idea to use standard library options wherever possible as long as there is no issue with needing support old compiler versions. I am not sure on the risk involved in doing this though. They may be some subtle differences in behavior.